ui.go

```diff
@@ -43,15 +42,31 @@ func newUI(w fyne.Window, p fyne.Preferences, host func() string) *ui {
 }
 
 func (u *ui) askShutdown() {
-	dialog.ShowConfirm("Shutdown", "Are you sure you want to shut down?",
-		func(ok bool) {
-			if !ok {
-				return
-			}
-
-			cmd := exec.Command("shutdown", "-h", "now")
-			_ = cmd.Start()
-		}, u.win)
+	var pup *widget.PopUp
+	shutdown := &widget.Button{Text: "Power Off", Importance: widget.HighImportance, OnTapped: func() {
+		cmd := exec.Command("shutdown", "-h", "now")
+		_ = cmd.Start()
+	}}
+	reboot := &widget.Button{Text: "Reboot", OnTapped: func() {
+		cmd := exec.Command("shutdown", "-r", "now")
+		_ = cmd.Start()
+	}}
+	cancel := &widget.Button{Text: "Cancel", OnTapped: func() {
+		pup.Hide()
+	}}
+	spacer := canvas.NewRectangle(color.Transparent)
+	spacer.SetMinSize(fyne.NewSize(0, 10))
+	buttons := container.NewPadded(
+		container.NewVBox(
+			shutdown,
+			reboot,
+			spacer,
+			widget.NewSeparator(),
+			spacer,
+			cancel,
```

Comment:
This cancel option makes sense - but the button flow is significantly different to the standard layout of, for example, a dialog - would it not be better to use a horizontal flow where the cancel is left-most like in other places?

Reply:
To me, it ended up looking dumb because the buttons were different sizes.